src/components/common/Swap/components/CoinsListModal/CoinsListModal.tsx

```diff
@@ -114,13 +142,14 @@ const CoinsListModal = ({ selectCoin, balances, verifiedAssetsOnly }: Props) =>
             onClick={() => selectCoin(value)}
           />
         )}
-        {sortedCoinsList.map(({ assetId }) => (
+        {sortedCoinsList.map(({assetId}) => (
           <div
             className={styles.tokenListItem}
             onClick={() => selectCoin(assetId)}
             key={assetId}
           >
             <CoinListItem
+              verifiedAssetData={verifiedAssetData}
```

Comment:
Instead of having to pass this array into each instance, theCoinListItem could include the useVerifiedAssets hook directly

Reply:
Done!